test/y18n-test.js

```diff
@@ -72,17 +102,56 @@ describe('y18n', function () {
 
       it('writes new word to locale file if updateFiles is true', function (done) {
         var __ = y18n({
-          locale: 'fr',
+          locale: 'fr_FR',
           directory: __dirname + '/locales'
         }).__
 
         __('banana', function (err) {
+          var locale = JSON.parse(fs.readFileSync('./test/locales/fr_FR.json', 'utf-8'))
+          locale.banana.should.equal('banana')
+          return done(err)
+        })
+      })
+
+      it('writes new word to language file if language_territory file does not exist', function (done) {
+        fs.writeFileSync('./test/locales/fr.json', '{"meow": "le meow"}', 'utf-8')
+
+        var __ = y18n({
+          locale: 'fr_FR',
+          directory: __dirname + '/locales'
+        }).__
+
+        __('meow').should.equal('le meow')
+        __('banana', function (err) {
           var locale = JSON.parse(fs.readFileSync('./test/locales/fr.json', 'utf-8'))
           locale.banana.should.equal('banana')
           return done(err)
         })
       })
 
+      it('writes word to missing locale file, if no fallback takes place', function (done) {
+        fs.writeFileSync('./test/locales/fr.json', '{"meow": "le meow"}', 'utf-8')
+
+        var __ = y18n({
+          locale: 'fr_FR',
+          fallbackToLanguage: false,
+          directory: __dirname + '/locales'
+        }).__
+
+        var frJson = JSON.parse(fs.readFileSync('./test/locales/fr.json', 'utf-8'))
+        frJson.should.deep.equal({
+          meow: 'le meow'
+        })
```

Comment:
I understand your confusion now - this doesn't make sense here. It should go in the callback function below. The point is to verify that the `fr.json` file remained untouched.

Sorry, I will fix.